Added docs for "functions_to_trace" option. (#6520)
getsentry/sentry-docs: src/platforms/common/configuration/options.mdx

</ConfigKey>

<ConfigKey name="functions-to-trace" supported={["python"]}>

> An optional list of functions that should be set up for performance monitoring. For each function in the list a span will be created when the function is executed.

Reviewer: ```suggestion
An optional list of functions that should be set up for performance monitoring. For each function in the list, a span will be created when the function is executed.
```

Functions in the list are represented as strings containing the fully qualified name of the fuction.

> This is a convenience option to not have manual instrumentation scattered all over your code base, but have one central place to configure what functions to trace.

Reviewer: ```suggestion
This is a convenient option, making it possible to have one central place for configuring what functions to trace, instead of having manual instrumentation scattered all over your code base.
```

> To learn more see the [Custom Instrumentation](/platforms/python/performance/instrumentation/custom-instrumentation/#define-span-creation-in-a-central-place) documentation.

Reviewer: ```suggestion
To learn more, see the [Custom Instrumentation](/platforms/python/performance/instrumentation/custom-instrumentation/#define-span-creation-in-a-central-place) documentation.
```
